packages/flutter_tools/test/general.shard/android/android_studio_test.dart

```diff
@@ -96,6 +109,71 @@ void main() {
       );
     });
 
+    testUsingContext('Can discover Android Studio >=4.1 location on Mac', () {
```

Comment:
This is kind of a nit, but the spacing in these tests makes them a bit hard to read. I would generally split up tests into 3 sections:

1. creating objects/mocks and setting up state
2. performing the action to be tested
3. making expectations about the results

Reply:
> This is kind of a nit, but the spacing in these tests makes them a bit hard to read. I would generally split up tests into 3 sections:
> 
> 1. creating objects/mocks and setting up state
> 2. performing the action to be tested
> 3. making expectations about the results

Hi @jonahwilliams ,

The devil is in the detail. I really sorry for having overlooked the code style.
Thank you very much for your suggestion.It's this kind of suggestion that I strive for to help me keep improving.

Best wishes!😁